docs/book/theme/index.hbs

```diff
@@ -139,6 +139,11 @@
                             <i id="git-repository-button" class="fa {{git_repository_icon}}"></i>
                         </a>
                         {{/if}}
+                        {{#if git_repository_edit_url}}
+                            <a href="{{git_repository_edit_url}}" title="Suggest an edit" aria-label="Suggest an edit">
+                                <i id="git-edit-button" class="fa fa-edit"></i>
```

Comment:
Is this change part of the scope of this PR?
Could we create a PR for add this option? 

Ideally, we should have 1 pr to address each need so that we can revert if we need it, it has less back and forward and make easier the review process.